packages/react/src/components/Tab/Tab-test.js

```diff
@@ -63,6 +63,18 @@ describe('Tab', () => {
       );
     });
 
+    it('has aria-disabled that matches disabled', () => {
```

Comment:
Do we need to test that the component is uninteractive when `disabled` is set? Meaning a user cannot tab/interact with it?

Reply:
How do I test that?

Reply:
Might require some of the helpers we've been talking about or added already, but maybe something like:

```jsx
import { getByText, isElementVisible } from '@carbon/test-utils/dom';
import { render, cleanup } from '@carbon/test-utils/react';
import {
  pressEnter,
  pressSpace,
  pressTab,
  pressShiftTab,
} from '@carbon/test-utils/keyboard';
import { Simulate } from 'react-dom/test-utils';

it('should not allow keyboard navigation to a disabled tab', () => {
  const { container } = render(
    <Tabs>
      <Tab label="Tab 1">
        Tab 1 content
      </Tab>
      <Tab label="Tab 2" disabled>
        Tab 2 content
      </Tab>
      <Tab label="Tab 3">
        Tab 3 content
      </Tab>
    </Tabs>
  );

  const tabs = [
    getByText('Tab 1'),
    getByText('Tab 2'),
    getByText('Tab 3'),
  ];

  pressTab();

  // Assert that first tab is focused after pressing tab
  expect(document.activeElement).toEqual(tabs[0]);

  pressTab();

  // Assert that the last tab is focused after pressing tab,
  // not the second tab
  expect(document.activeElement).toEqual(tabs[2]);
});

it('should not allow interaction with a disabled tab', () => {
  const { container } = render(
    <Tabs>
      <Tab label="Tab 1">
        Tab 1 content
      </Tab>
      <Tab label="Tab 2" disabled>
        Tab 2 content
      </Tab>
      <Tab label="Tab 3">
        Tab 3 content
      </Tab>
    </Tabs>
  );

  const tab = getByText('Tab 2');
  const tabContent = getByText('Tab 2 content');

  // Force focus even though this wouldn't happen in practice
  // to verify mouse and keyboard events don't trigger anything
  const interactions = [
    pressEnter(),
    pressSpace(),
    () => {
      Simulate.click(tab);
    },
  ];

  for (const interaction of interactions) {
    interaction();
    expect(isElementVisible(tabContent)).toBe(false);
  }
});
```